requirements-deploy.txt

```diff
@@ -3,24 +3,23 @@ appnope==0.1.3; sys_platform == "darwin" and python_version >= "3.8" and platfor
 argon2-cffi-bindings==21.2.0; python_version >= "3.7"
 argon2-cffi==21.3.0; python_version >= "3.7"
 argue==0.3.1
-astor==0.8.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
```

Comment:
Is astor no longer in use, or was this accidental deletion?

Reply:
It was a dependency of something else I believe. Potentially streamlit, which I guess would have dropped the dependency with the streamlit version upgrade.